packages/frontend-core/src/components/grid/lib/utils.ts

```diff
@@ -14,7 +14,9 @@ export const getCellID = (rowId: string, fieldName: string) => {
   return `${rowId}${CellIDSeparator}${fieldName}`
 }
 
-export const parseEventLocation = (e: MouseEvent & TouchEvent) => {
+export const parseEventLocation = (event: MouseEvent | TouchEvent) => {
+  const e = event as MouseEvent & TouchEvent
```

Comment:
Mixing params with actual usage. The current code uses `clientX` to detect what type is it. We could add some guard typing, but this does the job for now without modifying the executable code